components/HeroBanner/index.vue

```diff
@@ -2,14 +2,16 @@
   <div id="hero-banner" class="hero-banner">
     <div class="hero-banner__main">
       <div class="relative px-4 sm:pl-[15.5%]">
-        <h3 class="hero-banner__title">
-          Judul Landing Page
-        </h3>
-        <p class="hero-banner__desc">
-          Tulisan ini berisi deskripsi singkat dari judul yang terdapat pada judul utama.
-        </p>
+        <div class="hero-banner__box-title">
+          <h3 class="hero-banner__title">
+            Saatnya semua menikmati
+          </h3>
+          <h3 class="hero-banner__title">
+            manfaat teknonlogi
```

Comment:
I think there is a typo on the Copywrite @doohanas

Reply:
Oke kak, it's done to fix the typo ✌🏽 @Ibwedagama